test/apparmor.bats

```diff
@@ -27,7 +27,8 @@ load_default_apparmor_profile_and_run_a_container_with_it() {
 	setup_test
 	start_crio
 
-	sed -e 's/%VALUE%/runtime\/default/g' "$TESTDATA"/sandbox_config_apparmor.json > "$TESTDIR"/apparmor1.json
```

Comment:
based on what I see
```diff
[kir@kir-rhat cri-o]$ diff -u <(jq < test/testdata/sandbox_config.json) <(jq <test/testdata/sandbox_config_apparmor.json)
--- /dev/fd/63	2020-10-26 12:19:47.271341050 -0700
+++ /dev/fd/62	2020-10-26 12:19:47.272341064 -0700
@@ -7,7 +7,11 @@
   },
   "hostname": "crictl_host",
   "log_directory": "",
-  "dns_config": {
+  "dns_options": {
+    "servers": [
+      "server1.redhat.com",
+      "server2.redhat.com"
+    ],
     "searches": [
       "8.8.8.8"
     ]
@@ -27,12 +31,12 @@
     "group": "test"
   },
   "annotations": {
-    "owner": "hmeng",
-    "security.alpha.kubernetes.io/seccomp/pod": "unconfined"
+    "owner": "hmeng"
   },
   "linux": {
     "cgroup_parent": "pod_123-456.slice",
     "security_context": {
+      "apparmor_profile": "%VALUE%",
       "namespace_options": {
         "network": 0,
         "pid": 1,
```

I think you can just use `test/testdata/sandbox_config.json` and remove `test/testdata/sandbox_config_apparmor.json` for good.

Reply:
> I think you can just use test/testdata/sandbox_config.json and remove test/testdata/sandbox_config_apparmor.json for good.

Sorry for my brevity; what I meant here is not "you can use sandbox_config.json as-is" but "you can use sandbox_config.json as jq input, and set the proper `apparmor_profile`.

Reply:
fixed!